tests/unit/ml/test_sql.py

```diff
@@ -95,6 +95,15 @@ def test_min_max_scaler_produces_correct_sql(
     assert sql == "ML.MIN_MAX_SCALER(col_a) OVER() AS scaled_col_a"
 
 
+def test__k_bind_discretizer_produces_correct_sql(
+    base_sql_generator: ml_sql.BaseSqlGenerator,
+):
+    sql = base_sql_generator.ml_k_bind_discretizer(
+        "col_a", [1, 2, 3, 4], "scaled_col_a"  # type:ignore
```

Comment:
Please refrain from using type:ignore, unless we know and document it why. For this one, it is the function signature doesn't match. See the other comment.

Reply:
Resolved.